src/components/Homepage/LendByCategory/LoanCategory.vue

```diff
@@ -346,7 +346,7 @@ $card-half-space: rem-calc(14/2);
 }
 
 .cards-mg-promo {
-	height: calc(100% - 2rem);
+	height: calc(100% - 1.35rem);
```

Comment:
I have a different approach to getting these to line up using flexbox that I'm pushing up now.

Reply:
@ryan awesome, i'll take your changes